client/themes/morning.css

```diff
@@ -135,25 +128,6 @@
 	color: #b7c5d1;
```

Comment:
This and `#chat table.channel-list td` also use this color, are we not able to use `var` here?

Reply:
I've done a couple things in 8c09be9d, let me know what you think:

- For unhandled messages, I moved the override to main styling and used muted color (I also applied it to the icons for MOTD and CTCP). I removed color for `#chat .from` which should always be set for specific things: colored or uncolored nicknames, icons, etc.
- Removed color overrides for the channel list. There was no real reason to override the main color so now channel list table looks the same as ignored/ban user tables. I also removed the override on channel names in this table because I didn't think it brings anything to do that, and it's now more consistent with inline channels in chat content, and would keep it consistent if at some point we decided to style inline channels.

I didn't touch `#windows .window h2` because that's something I'd want to do in the main style without resorting to this specific variable (so have a new variable for headings for example) and that's way outside of what I was trying to achieve here, I'll deal with this later if that works for you.